core-bundle/src/Resources/config/services.yml

```diff
@@ -1078,13 +1077,13 @@ services:
             version: 4.13
             message: Using the "%alias_id%" service ID has been deprecated and will no longer work in Contao 5.0. Please use "contao.image.sizes" instead.
 
-    contao.image.resizer:
-        alias: contao.image.legacy_resizer
+    contao.image.legacy_resizer:
```

Comment:
Why keep this at all?

Reply:
For BC? This service is pretty probable to get decorated IMO.

Reply:
But we are allowed to break BC in Contao 5.0. Still want to keep the old service ID?

Reply:
I’d vote for keeping the old service ID. But I’m fine either way.